lessons/Algebra2/exploring-linear-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,265 @@
+
+Let students discuss, and explain their thinking. If possible, mark off a single point for each of the hypothetical percentages, then connect those points to show a straight line. Note that some of these new points would require changing the x- and y-axes of our display!
+
+When we see patterns in data, we can use those patterns to __make predictions__ based on that data. We can even draw a line to show all the possible predictions at once! These predictions represent our "best guess" at the underlying relationship in the data, as we try to model that relationship using math.
+
+These models are just functions being graphed on top of the scatter plot, with the goal of minimizing the distance between the line and all the points on the plot. For straight-line relationships, these are _linear functions_ or "linear models". The straight-line graph of these models is sometimes called the "regression line" or the "predictor function", but you may have heard it called the @vocab{line of best fit}.
+
+When we make a model, we want it to be the closest possible approximation of all the points. A "good fit" has most of the points very close to the line, and a "bad fit" has the points very far away.
+
+You may already be familiar with the different forms of linear models:
+
+[cols="^1,^1,^1", options="header"]
+|===
+| Standard Form 	| Point-Slope Form			| Slope-Intercept Form
+| @math{Ax+By = C}	| @math{y-y_1 = m(x-x_1)}	| @math{y = mx+b}
+|===
+
+Depending on what part of the model we care about most, we might choose to use one display over another.
+
+- Standard Form makes it easy to find the x- and y-intercepts of the line
+- Point-Slope Form makes it easy to find the equation of the line given a single point and slope
+- Slope-Intercept Form makes it really easy to read off the slope and y-intercept
+
+Since we're looking for both the slope and the y-intercept for this model, it makes sense to use Slope-Intercept form.
+
+
+=== Investigate
+
+[.lesson-instruction]
+- Complete the first section ("Build a Model from Samples") in @printable-exercise{model-college-v-income.adoc}.
+- How well did your model work for Alabama and Alaska? Why didn't it work as well for other states?
+- How can we measure "how well a model fits"?
+
+Before students complete the remainder of the workbook page, confirm that they were able to successfully compute slope and y-intercept, define and test `f(x)` in Pyret, and evaluate the predictive value of `f(x)`.
+
+Pyret includes a function called `fit-model`. Find this contract in your contracts pages! Like `scatter-plot`, it consumes columns for our labels, our @math{x}s and ours @math{y}s. However, it __also consumes a function!__ It produces a scatter plot, with the function graphed on top of it.
+
+[.lesson-instruction]
+- Complete the second section ("Fit the Model") in @printable-exercise{model-college-v-income.adoc}.
+- What does @math{R^2} mean?
```

Comment:
If we want to have students understand what R^2 means, I think we have to provide another page that actually focuses on this. Right now we are asking teachers to lecture without the images they would need to facilitate a conversation. It feels like a very heavy lift to have students doing this level of abstract thinking without examples. It's also confusing that we're asking the question in the lesson instruction and not that far after we offer 

> Deeper discussion of R^2 and least-squares regression may be appropriate for older students, or in a dedicated statistics class.

I'm unclear about the lesson goals.

Reply:
I don't think a deep understanding of R-squared is an important part of the goal. As long as students understand that it's measuring goodness-of-fit, that's enough for this curriculum.